functions/src/modules/CR.functions.ts

```diff
@@ -63,3 +70,149 @@ export const copySubmissionsToProcessing = functions.pubsub
 
     return true;
   });
+
+/**
+ * Gets lists with spare files
+ */
+export const getLists = functions.https.onCall(async (data, context) => {
+  if (!context.auth || !context.auth.token || !context.auth.token.coordinator) {
+    throw new functions.https.HttpsError(
+      'permission-denied',
+      'The function must be called by an authenticated coordinator.'
+    );
+  }
+
+  const gsheets = new GoogleSheets(
+    functions.config().cr.allotments.spreadsheet.id
+  );
+  const allotmentsSheet = await gsheets.useSheet(
+    ISoundQualityReportSheet.Allotments
+  );
+
+  const rows = await allotmentsSheet.getRows();
+
+  return rows
+    .filter(item => !item['Status'] && item['List'])
+    .map(item => item['List'])
+    .filter((value, index, self) => self.indexOf(value) === index);
+});
+
+/**
+ * Gets spare files for specified list and languages
+ */
+export const getSpareFiles = functions.https.onCall(
+  async ({ list, language, languages, count }, context) => {
```

Comment:
Allotment form that is going to call this function will be sending only `languages`, no need to maintain backward-compatibility with the old form that was sending only one `language`.